contracts/USM.sol

```diff
@@ -29,13 +27,28 @@ contract USM is ERC20 {
         ethPool = 0;
     }
 
+    /**
+     * @notice Multiplies x and y, assuming they are both fixed point with 27 digits.
+     */
+    function muld(uint256 x, uint256 y) internal pure returns (uint256) {
```

Comment:
Could we make `muld` and `divd` a little more descriptive?

Perhaps something like `mulRay` and `divRay`? (not entirely convinced by these either 😂 )

I'm open to ideas on this one.

Reply:
I've used `muld` (for Multiply Decimals) for ages, but I'm not emotionally attached to it. `mulf` (for Multiply Fixed point numbers) would be probably better. If you want to go for longer function names I would do `mulFixed` or `mulF27` over `mulRay`.

Reply:
I like `mulFixed`!